packages/react/src/file-upload/FileUploadExistingFileList.tsx

```diff
@@ -13,7 +13,7 @@ export const FileUploadExistingFileList = ({
 	hideThumbnails,
 	onRemove,
 }: FileUploadExistingFileListProps) => {
-	if (!files?.length) {
+	if (!files.length) {
 		return null;
 	}
```

Comment:
I think it's more readable/safer to conditionally render a component where it's used rather than within. I've seen situations where, unexpectedly, components are missing in the UI because it wasn't obvious that this conditional rendering was occurring. I see that this code wasn't written in this PR but it'd be nice to clean up this stuff as we go.

What do you think?

Reply:
Yeah, I can understand both camps. Perhaps we should decide and make a pass on the whole DS in another ticket?

Reply:
Ok, sounds like a good plan 🙌